frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -350,20 +360,23 @@ class EditEmail extends Component {
                 )}
               </label>
               <OverlayTrigger
-                trigger="focus"
+                trigger="click"
```

Comment:
Here, you should use `"focus"` and ... (see next comment)

Reply:
changed in ea161a4